### PR TITLE
feat(base/ui/web): added autofocus prop to button component

### PR DESCRIPTION
**Pull request by Calinteodor**
After we press the chat tab, chat input gets automatically focused. 
This isn't the case for when we press the polls tab. Create a Poll button doesn't get focused.

1. Added autoFocus prop to UI Button component.
2. Set autoFocus default prop to false.
3. Added autoFocus = true to Create a Poll button.

### CODE REVIEW

File: react/features/chat/components/web/Chat.js [hunk 1]
```diff
@@ -162,9 +162,7 @@ class Chat extends AbstractChat<Props> {
                     <MessageContainer
                         messages = { this.props._messages } />
                     <MessageRecipient />
-
                     <ChatInput
-                        onResize = { this._onChatInputResize }
```
Comment (saghul) [inline react/features/chat/components/web/Chat.js:167]: Why was this removed?

**Review by saghul — COMMENTED** [review]
Not sure I understand button auto-focus, is this for accessibility?

File: react/features/chat/components/web/Chat.js (hunk 1 above)
Comment (Calinteodor) [reply react/features/chat/components/web/Chat.js:167]: This method is not declared anywhere.

**Comment by Calinteodor** [comment]
> Not sure I understand button auto-focus, is this for accessibility?

Yes. When we press the polls tab the focus should move automatically to the create a poll button.

File: react/features/chat/components/web/Chat.js (hunk 1 above)
Comment (saghul) [reply react/features/chat/components/web/Chat.js:167]: LoL

**Review by saghul — APPROVED** [review]
